src/interface/types.ts

```diff
@@ -0,0 +1,9 @@
+'use strict'
```

Comment:
I suggest using the api-snippets solution based on`openapi-typescript` to automatically generate the interfaces from the YAML: example of the command: https://github.com/mojaloop/api-snippets/blob/master/package.json#L12
then you will have all interfaces in `openapi.ts` and only the need to re-export they left: https://github.com/mojaloop/api-snippets/blob/master/src/v1_1/index.ts

Reply:
I thought about it but since I don't forsee the interface changing and this interface being the only one that deviates from the ML API spec I didn't think it was worth the trouble.

Reply:
If you think the effort is worth it in the long run I can set it up.

Reply:
Can I also say how freaking nice it is to have the definitions pre-packaged and ready to go `import { Schemas } from '@mojaloop/api-snippets/lib/v1_1';`